hack/run-locally.sh

```diff
@@ -158,15 +164,20 @@ function create_cluster() {
 function print_usage() {
     >&2 echo "Usage: $0 [-c <cluster-dir>] [options]
 
-$0 detects whether the cluster given by the '-c' option does not yet exist or
+$0 runs against an existing cluster using the specific '-k' option or 
+detects whether the cluster given by the '-c' option does not yet exist or
 is already running. If it does not yet exist the cluster will be created using
 the options supplied below. If it is already running, any running
 cluster-network-operator will be stopped, plugin image overrides applied, and
 a local cluster-network-operator started.
 
 The following options are accepted for both new and existing clusters:
 
- -c DIR           the cluster installation temporary directory (can also be given via CLUSTER_DIR); required
+ -c DIR           the cluster installation temporary directory (can also be given via CLUSTER_DIR); 
+ -k KUBECONFIG    the kubeconfig pointing to the existing cluster (can also be given via KUBECONFIG); 
+
+-c or -k are required 
```

Comment:
"Either -c or -k is required"
or really
"Either -c or -k (or their corresponding environment variable) is required"